### PR TITLE
Device nokia rx51 modules

### PR DESCRIPTION
**Pull request by MartijnBraam**
This adds module loading support for the n900.

### CODE REVIEW

File: aports/linux-nokia-rx51/APKBUILD [hunk 1]
```diff
@@ -85,6 +85,7 @@ build() {
 	unset LDFLAGS
 	make ARCH="$_carch" CC="${CC:-gcc}" \
 		KBUILD_BUILD_VERSION="$((pkgrel + 1 ))-Alpine" \
+		CFLAGS_MODULE=-fno-pic \
```
Comment (ollieparanoid) [inline aports/linux-nokia-rx51/APKBUILD]: Why is that needed?

Comment (MartijnBraam) [inline aports/linux-nokia-rx51/APKBUILD]: This fixes loading modules with modprobe in the initramfs. The position indenpendant code breaks for some reason and this is the fix recommended all over the internet.
